### DISCUSSION TITLE
Issue with sending emails

### DISCUSSION THREAD

**Discussion by beltebelt** [pocketbase/pocketbase#4992] [Q&A]
Is there a difference in the way the test emails are sent and how `pb.collection("users").requestVerification(email)` works?

I have configured pocketbase to use my smtp server, and I am able to send the test emails. However I don't receive an email when are use `requestVerification` method from the sdk, even though I am getting the same 204 in both the cases.

**Comment by ganigeorgiev** [comment]
The only difference between the test emails action and the actual one is that the test is using a dummy user record.

If you are not receiving an email make sure that the user with the provided email exists. Note that you'll receive a 204 response even if there is no user with such email (this is to limit the users enumeration).

You can find more detailed errors in the Admin UI > Logs and/or start the executable with `--dev`

**Reply by beltebelt** [reply]
Sure. I think I am doing all of that. I am checking if the user exists and only requesting verification using the resulting record.email. Using `go run main.go server --dev` didn't result in any error as well. So I am not sure if it even has anything to do with pocketbase, just weird that it works for the test emails and not when I use the sdk's `requestVerification`. Another puzzling thing is that `requestPasswordReset` mail works just fine.

**Reply by beltebelt** [reply]
Tried changing the smtp settings from my self hosted smtp server to AWS SES (hoping that was the issue). But similar behaviour, test emails work, `requestPasswordReset` works, but `requestVerification` doesn't work

**Reply by ganigeorgiev** [reply]
I think this is going nowhere. Without clear reproducible steps I'm not sure how to help.

If you think that this a PocketBase issue please create a **minimal** reproducible repo and I'll try to have a look at it but until then I don't think anyone will be able to help you.

**Comment by Salman9000** [comment]
What settings did you use to send test emails? I am unable to send even those!

**Reply by ganigeorgiev** [reply]
@Salman9000 In order to send reliably emails you must configure the [SMTP settings](https://pocketbase.io/docs/going-to-production/#use-smtp-mail-server).

For test purposes I often use [mailtrap.io](https://mailtrap.io/) or [SMTP bucket](https://www.smtpbucket.com/) if you don't want to register.

If you need further help, please open a new Q&A discussion elaborating more on your current setup (you can also start the executable `--dev` for more detailed errors and/or check the logs in the _Admin UI > Logs_).
